plugins/hypervisors/xenserver/src/com/cloud/hypervisor/xenserver/resource/Xenserver625StorageProcessor.java

```diff
@@ -576,13 +578,20 @@ public Answer backupSnapshot(final CopyCommand cmd) {
             s_logger.info("New snapshot details: " + newSnapshot.toString());
             s_logger.info("New snapshot physical utilization: "+physicalSize);
 
+            successful = true;
+
             return new CopyCmdAnswer(newSnapshot);
         } catch (final Types.XenAPIException e) {
             details = "BackupSnapshot Failed due to " + e.toString();
             s_logger.warn(details, e);
         } catch (final Exception e) {
```

Comment:
It seems that this generic exception can be removed.

Reply:
Actually not, there are multiple other exceptions can be thrown which this block should catch them. The difference is that `Types.XenAPIException` doesn't have a meaningful `getMessage()` method. I'll combine them together though.